raiden-ts/src/path/epics.ts

```diff
@@ -63,21 +84,46 @@ export const pathFindServiceEpic = (
               // else, if possible, use a direct transfer
               else if (
                 channelCanRoute(state, presences, tokenNetwork, target, action.meta.value) === true
-              )
+              ) {
                 return of([{ path: [state.address, target], fee: Zero as Int<32> }]);
-              // else, request a route from PFS
-              else if (pfs !== null) {
-                // from all channels
-                return fromFetch(`${pfs}/api/v1/${tokenNetwork}/paths`, {
-                  method: 'POST',
-                  headers: { 'Content-Type': 'application/json' },
-                  body: losslessStringify({
-                    from: address,
-                    to: target,
-                    value: UInt(32).encode(action.meta.value),
-                    max_paths: 10,
-                  }),
-                }).pipe(
+              } else if (!action.payload.pfs && configPfs === null) {
+                // pfs not specified in action and disabled (null) in config
+                throw new Error(`PFS disabled and no direct route available`);
+              } else {
+                // else, request a route from PFS.
+                // pfs$ - Observable which emits one PFS info and then completes
+                const pfs$ = action.payload.pfs
+                  ? // first, honor action.payload.pfs
+                    of(action.payload.pfs)
+                  : !isNil(configPfs)
```

Comment:
`isNil` actually test if `null` || `undefined`. It stayed because a previous version used it in some more places, including over result of function call, and to do the double check without the helper I'd need to store it in a variable to test, which would be troublesome in the `if`. Now, it's simple enough to be removed, although I think it's already used somewhere else so wouldn't save space anyway.

Reply:
`foo == null` also tests `null || undefined` and `isNil` actually uses `==` [under the hood](https://github.com/lodash/lodash/blob/master/isNil.js) as well. Doesn't seem to be used on `lib2` branch if i search for it

Reply:
Done in #497